notes/goals.json

```
@@ -1124,7 +1124,7 @@
   {
     "missionNumber": 9,
     "goalName": "GetIntoSubPen",
-    "goalText": "Enter Main building on north side of Dockyard. There may be more than one entry point.",
+    "goalText": "Enter Main building on north side of Dockyard.  There may be more than one entry point.",
```

looks like this should be a single space

```

[GOAL: GetIntoSubPen (Primary)]
Enter Main building on north side of Dockyard. There may be more than one entry point.

```
```suggestion
    "goalText": "Enter Main building on north side of Dockyard. There may be more than one entry point.",
```

---

Right, goals.txt and goals.json shouldn't be getting changed.